backend/kesaseteli/applications/tests/test_youth_applications_api.py

```diff
@@ -414,6 +414,7 @@ def test_youth_applications_process_valid_pk(
         )
 
 
+@override_settings(NEXT_PUBLIC_DISABLE_VTJ=True)
```

Comment:
Why this? Something related has changed?

Reply:
This function's test cases using staff_client/superuser_client and NEXT_PUBLIC_MOCK_FLAG=False [weren't passing](https://github.com/City-of-Helsinki/yjdh/actions/runs/3639752757/jobs/6143534465) so I added this, I would think it's not related to any code changes in this pull request i.e. that these cases should've been broken earlier already.

It's because of [fetch_vtj_json](https://github.com/City-of-Helsinki/yjdh/blob/5f932b6c25499f9c8ef8aa7827e72b612a6c9573/backend/kesaseteli/applications/models.py#L267) being called which needs NEXT_PUBLIC_DISABLE_VTJ=True or NEXT_PUBLIC_MOCK_FLAG=True to not use real VTJ integration and when NEXT_PUBLIC_MOCK_FLAG=False and ks-review.yml has [NEXT_PUBLIC_DISABLE_VTJ: 0](https://github.com/City-of-Helsinki/yjdh/blob/5f932b6c25499f9c8ef8aa7827e72b612a6c9573/.github/workflows/ks-review.yml#L46) the test cases used real VTJ integration which failed in review environment because the VTJ integration wasn't properly configured there. IMHO it's a good idea not to use real VTJ integration in this test so ensuring that it is not used is in my opinion good.